read.go

```diff
@@ -30,6 +30,7 @@ var (
 	// dataset returned is still valid.
 	ErrorUnsupportedBitsAllocated = errors.New("unsupported BitsAllocated")
 	errorUnableToParseFloat       = errors.New("unable to parse float type")
+	ErrorDoesNotConformToDICOM    = errors.New("field length is not even, in violation of DICOM spec")
```

Comment:
can we rename this error to something more specific to the evenness check?

```suggestion
	ErrorExpectedEvenLength    = errors.New("field length is not even, in violation of DICOM spec")
```